dice-game-ui-angular/src/app/start-game/start-game.component.ts

```diff
@@ -11,24 +11,30 @@ import { StatisticsService } from '../services/statistics-service';
   styleUrls: ['./start-game.component.scss'],
 })
 export class StartGameComponent implements OnInit {
-  public size: number;
-  public sizeOptions: number[] = [];
+  private readonly localPlayer = 'local player';
```

Comment:
Private members should be located after all public and protected members of same type.

Reply:
Fixed